test/integration/object_store/create_email_test.rb

```diff
@@ -0,0 +1,32 @@
+require "test_helper"
+require "capybara/rails"
+
+class CreateEmailTest < ActionDispatch::IntegrationTest
```

Comment:
I read this and mistook it for creating/sending an email. Maybe worth renaming to `create_email_content_item_test.rb` or `create_email_address_test.rb`.